src/buffer.h

```diff
@@ -150,6 +150,11 @@ class Buffer {
   /// initial count. This requires the format to have been set.
   void ResizeTo(uint32_t element_count);
```

Comment:
Maybe we should file a bug to rename this `SetElementCount` or `SetSizeToCountElements` or something ? ...

Reply:
There is a SetElementCount already:
  /// Sets the number of elements in the buffer.
  void SetElementCount(uint32_t count) { element_count_ = count; }
I filed issue #597